File: javascript/3170.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var b = parseInt(lines.shift());
var g = Math.floor(lines.shift() / 2);
var f = g - b;
if (b > g) {
     console.log('Amelia tem todas bolinhas!')
} else if (!f) {
     console.log('Amelia tem todas bolinhas!')
} else {
     console.log(`Faltam ${g - b} bolinha(s)`)
}

File: C++/3170.cpp
#include <iostream>

using namespace std;

int main() {
    int b, g, f;
    cin >> b >> g;
    f = g / 2 - b;
    
    if(f > 0) {
        cout << "Faltam " << f << " bolinha(s)";
    }else {
        cout << "Amelia tem todas bolinhas!";
    }
    cout << endl;
    return 0;
}